media/css/firefox/template-resp.less

```diff
@@ -24,6 +24,10 @@
     display: block;
 }
 
+.stacked h4 {
+    line-height: 1.3;
+}
```

Comment:
Somehow `firefox/template-resp.less` used on this page, and adding this rule fixes the `line-height` on the Firefox ESR page as well so that's fine. The same rule was added `firefox/template.less` for consistency.